arch/sim/src/sim/sim_checknuttxtypes.c

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
+ * arch/sim/src/sim/sim_checknuttxtypes.c
```

Comment:
should we move the check to fs/hostfs/hostfs.c?

Reply:
No, because this is very specific to sim.

Reply:
Ok, let's change to sim_checkhostfs.c?

Reply:
renamed to sim_checkhostfstypes.c